### ISSUE TITLE
what is the next step or plan for this repo?

### ISSUE THREAD

**Issue by zimenglan-sysu-512** [facebookresearch/maskrcnn-benchmark#551]
## ❓ Questions and Help
hi @fmassa,
if u don't mind, can u share some further plans or any big change recently for this repo ?
thanks

**Comment by fmassa** [comment]
Hi,

This repo is a reference implementation for Faster / Mask R-CNN, that other people can leverage to bootstrap their research.
I have already pushed all the major elements that I was planning for this repo.
I'll now be spending most of my time improving torchvision

**Comment by qianyizhang** [comment]
in this case, whats the next step for torchvision :P
is there any plan to accelerate augmentation with GPU (color jittering, rotation, etc)?

**Comment by zimenglan-sysu-512** [comment]
hi @fmassa 
thanks for your great work. but do u still have a plan to merge the `optimized` branch to `master` branch to support `fp16` and multi-machines-multi-gpus training/inference?
thanks.

**Comment by fmassa** [comment]
@qianyizhang yes, performing data augmentation on the GPU (by leveraging native pytorch operators) is on the roadmap, but some other things need to be sorted out before.
I should release a roadmap for torchvision soon in the repo.

@zimenglan-sysu-512 about merging the `optimized` branch, I haven't tested it myself. From my understanding it should be straightforward to run it and use it, but given that I haven't had the time to look into it, I won't be merging it before some more testing.
But if the community uses it and has no problem with it, let me know and I'll happily merge it!

**Comment by koenvandesande** [comment]
Can you clarify the state of the keypoint detection? The model zoo has only a single entry (compared to Detectron which has more). Does it work for training, or only for inference?

**Comment by fmassa** [comment]
@koenvandesande Keypoints work for training, and give the expected results as Detectron, see https://github.com/facebookresearch/maskrcnn-benchmark/pull/69

**Comment by TyroneLi** [comment]
Will facebook AI add some research project in this repo just like tensorflow models repo?
